shared/scss/_tracker-network-icons.scss

```diff
@@ -1,279 +1,33 @@
-
-$icon__tn__criteo--chrome: "chrome-extension://__MSG_@@extension_id__/img/tn-criteo.svg";
-$icon__tn__criteo--moz: "moz-extension://__MSG_@@extension_id__/img/tn-criteo.svg";
-
-$icon__tn__facebook--chrome: "chrome-extension://__MSG_@@extension_id__/img/tn-facebook.svg";
-$icon__tn__facebook--moz: "moz-extension://__MSG_@@extension_id__/img/tn-facebook.svg";
-
-$icon__tn__fox--chrome: "chrome-extension://__MSG_@@extension_id__/img/tn-fox.svg";
-$icon__tn__fox--moz: "moz-extension://__MSG_@@extension_id__/img/tn-fox.svg";
-
-$icon__tn__google--chrome: "chrome-extension://__MSG_@@extension_id__/img/tn-google.svg";
-$icon__tn__google--moz: "moz-extension://__MSG_@@extension_id__/img/tn-google.svg";
-
-$icon__tn__mixpanel--chrome: "chrome-extension://__MSG_@@extension_id__/img/tn-mixpanel.svg";
-$icon__tn__mixpanel--moz: "moz-extension://__MSG_@@extension_id__/img/tn-mixpanel.svg";
-
-$icon__tn__moat--chrome: "chrome-extension://__MSG_@@extension_id__/img/tn-moat.svg";
-$icon__tn__moat--moz: "moz-extension://__MSG_@@extension_id__/img/tn-moat.svg";
-
-$icon__tn__newrelic--chrome: "chrome-extension://__MSG_@@extension_id__/img/tn-newrelic.svg";
-$icon__tn__newrelic--moz: "moz-extension://__MSG_@@extension_id__/img/tn-newrelic.svg";
-
-$icon__tn__nielsen--chrome: "chrome-extension://__MSG_@@extension_id__/img/tn-nielsen.svg";
-$icon__tn__nielsen--moz: "moz-extension://__MSG_@@extension_id__/img/tn-nielsen.svg";
-
-$icon__tn__openx--chrome: "chrome-extension://__MSG_@@extension_id__/img/tn-openx.svg";
-$icon__tn__openx--moz: "moz-extension://__MSG_@@extension_id__/img/tn-openx.svg";
-
-$icon__tn__outbrain--chrome: "chrome-extension://__MSG_@@extension_id__/img/tn-outbrain.svg";
-$icon__tn__outbrain--moz: "moz-extension://__MSG_@@extension_id__/img/tn-outbrain.svg";
-
-$icon__tn__quantcast--chrome: "chrome-extension://__MSG_@@extension_id__/img/tn-quantcast.svg";
-$icon__tn__quantcast--moz: "moz-extension://__MSG_@@extension_id__/img/tn-quantcast.svg";
-
-$icon__tn__salesforce--chrome: "chrome-extension://__MSG_@@extension_id__/img/tn-salesforce.svg";
-$icon__tn__salesforce--moz: "moz-extension://__MSG_@@extension_id__/img/tn-salesforce.svg";
-
-$icon__tn__taboola--chrome: "chrome-extension://__MSG_@@extension_id__/img/tn-taboola.svg";
-$icon__tn__taboola--moz: "moz-extension://__MSG_@@extension_id__/img/tn-taboola.svg";
-
-$icon__tn__twitter--chrome: "chrome-extension://__MSG_@@extension_id__/img/tn-twitter.svg";
-$icon__tn__twitter--moz: "moz-extension://__MSG_@@extension_id__/img/tn-twitter.svg";
-
-$icon__tn__yandex--chrome: "chrome-extension://__MSG_@@extension_id__/img/tn-yandex.svg";
-$icon__tn__yandex--moz: "moz-extension://__MSG_@@extension_id__/img/tn-yandex.svg";
+$icons: ("adobe", "adtech", "amazon", "aol", "appnexus", "casale", "chartbeat", "comscore", "criteo", "facebook", "fox", "generic", "google", "mixpanel", "moat", "newrelic", "nielsen", "openx", "outbrain", "quantcast", "salesforce", "taboola", "twitter", "yandex");
```

Comment:
@laurengarcia I had to modify the styles for the trackers subview anyway, because of the new icons (otherwise they'd be totally transparent there).

The generic icon only makes sense for the pills ux in the main view, and also I didn't want to change how the trackers subview looks, so I thought of adding a new mixin here to add a bg color, grey, only when we know we have an icon to render for that site.

So my options at these point were:
- write this mixin in a really verbose way, like the rest of this file, writing the same rule over and over again but with different class names
- I could also have written a script to do that
- Have a list of available icon names through which I can iterate, and while I'm at it, make the rest of the file less verbose as well.

I didn't use the technical design template because:
- I'm on a strict time table (hoping to get this merged tomorrow)
- This is arguably better code
- It's easy to test: either it renders the icons correctly or it doesn't

Let me know if you want to discuss this further!